src/Core/WinSWCore/Configuration/YamlConfiguration.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+
+namespace winsw.Configuration
+{
+    public class YamlConfiguration
+    {
+        public string? id;
+        public string? name;
+        public string? description;
+        public string? executable;
+        public string? workingdirectory;
+
+        public ServiceAccount? serviceaccount;
+        public Log? log;
+
+        public List<Downloading>? download;
+    }
+
+    public class ServiceAccount
+    {
+        public string name;
+        public string domain;
+        public string user;
+        public string allowservicelogon;
+    }
+
+    public class Log
+    {
+        public string? sizeThreshold;
+        public string? keepFiles;
+        public string? pattern;
+        public string? autoRollAtTime;
+        public string? period;
+        public string? mod;
+    }
+
+    public class Downloading
```

Comment:
Could it be just replaced by https://github.com/winsw/winsw/blob/70d83c045cfaabe9cb85e471c2972935698006c2/src/Core/WinSWCore/Download.cs ?

Reply:
Its possible. I will. Thank you for mention !